source/_patterns/02-molecules/blocks/hero/_hero.scss

```diff
@@ -7,10 +7,6 @@
   }
 }
 
-.jcc-hero--viewport-height {
-  @include u-minh("viewport");
-}
-
```

Comment:
I think I added this for the forms lookup page, which is essentially a full-height hero component. Did you remove because it was breaking something?

Reply:
restored